SQLite/Core/Connection.swift

```diff
@@ -22,6 +22,7 @@
 // THE SOFTWARE.
 //
 
+import Foundation
```

Comment:
That would be an option. In case of Foundation I wouldn't think that's necessary, it's highly likely all of Foundation are imported elsewhere anyway.